### PR TITLE
Update README to show setup instrcutions in main README

### LINKED ISSUE #108: Update README
Update README to show the setup instructions in the main README itself

### PR DESCRIPTION
**Pull request by SaurabhAgarwala**
Fixes #108 

This PR updates the README to show complete setup instructions in the main README.

Also, auto-formatter makes some recommended changes in the README styling

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,56 +1,109 @@
...
+  - Gunicorn
+  - Google Cloud Platform
+  - Python Anywhere
+  - Netlify
 
 ## Live Demo
- - [The App](https://deploy-preview-90--fellowship-companion.netlify.app/)
- - [API Endpoints](https://agsaurabh.pythonanywhere.com/api/v1/overview/) (docs comming soon)
+
+- [The App](https://deploy-preview-90--fellowship-companion.netlify.app/)
```
Comment (AlbertoPdRF) [inline README.md]:
```suggestion
- [The App](https://fellowship-companion.netlify.app/)
```

This is the correct URL, the other one is just for a preview deploy.

**Review by LakshyaKhatri — APPROVED** [review]
:tada:

File: README.md [hunk 2]
```diff
@@ -1,56 +1,109 @@
...
+- Map for showing country-wise contributions
+- UI for Popular Languages and Repositories
+- UI improvements
 
 ## Authors
+
 | <img src="https://github.com/AlbertoPdRF.png" alt="Alberto Pérez de Rada Fiol" width="100" height="100" /> | <img src="https://github.com/LakshyaKhatri.png" alt="Lakshya Khatri" width="100" height="100" /> | <img src="https://github.com/SaurabhAgarwala.png" alt="Saurabh Agarwala" width="100" height="100" /> |
-| ----- | ----- | ----- |
-| [Alberto Pérez de Rada Fiol](https://github.com/AlbertoPdRF) | [Lakshya Khatri](https://github.com/LakshyaKhatri) | [Saurabh Agarwala](https://github.com/SaurabhAgarwala) |
+| ---------------------------------------------------------------------------------------------------------- | ------------------------------------------------------------------------------------------------ | ---------------------------------------------------------------------------------------------------- |
```
Comment (AlbertoPdRF) [inline README.md:104]: I'm not sure where all of this comes from, but if it looks good, then it's fine!

**Review by AlbertoPdRF — CHANGES_REQUESTED** [review]
There's one thing to change, I've made a suggestion!

File: README.md (hunk 2 above)
Comment (SaurabhAgarwala) [reply README.md:104]: It is done through auto-formatter. I check it, looks good

**Pull request merged**